src/commands/sensor/commands/deletedatastream.handler.ts

```diff
@@ -19,7 +19,7 @@ export class DeleteDataStreamCommandHandler implements ICommandHandler<DeleteDat
     }
 
     const aggregate = this.publisher.mergeObjectContext(sensorAggregate);
-    aggregate.deleteDataStream(command.name);
+    aggregate.deleteDataStream(command.id, command.dataStreamId);
```

Comment:
Is het duidelijker om hier de `state.sensorId` in te stoppen ipv `command.id`? Het is mooi dat het id ook in het command zit, maar bij het produceren van events gaat het om de aggregate id, right?